tests/checks/integration/test_elastic.py

```diff
@@ -297,3 +350,86 @@ def test_health_event(self):
             tags=['host:localhost', 'port:9200'],
             count=1
         )
+
+    def test_pshard_metrics(self):
+        """ Tests that the pshard related metrics are forwarded  """
+
+        config = {'instances': [
+            {'url': 'http://localhost:9200', 'pshard_stats': True}
+        ]}
+        # Cleaning up everything won't hurt.
+        requests.delete('http://localhost:9200/_all/')
+
+        requests.put('http://localhost:9200/testindex/',
+                     data='{"settings": {"index": {"number_of_replicas": 1,'
+                          '"number_of_shards":5}}}')
+        requests.put('http://localhost:9200/testindex/testtype/2',
+                     data='{"name": "Jane Doe", "age": 27}')
+        requests.put('http://localhost:9200/testindex/testtype/1',
+                     data='{"name": "John Doe", "age": 42}')
+
+        time.sleep(ELASTIC_LATENCY)
```

Comment:
Yeah it seems like I found exactly what I needed on this page: https://www.elastic.co/guide/en/elasticsearch/guide/current/translog.html. I'll try that out ASAP.